osu.Game/Screens/Play/SongProgress.cs

```diff
@@ -42,7 +42,8 @@ public class SongProgress : OverlayContainer, ISkinnableDrawable
         /// </summary>
         public readonly Bindable<bool> AllowSeeking = new Bindable<bool>();
 
-        public readonly Bindable<bool> ShowGraph = new Bindable<bool>();
+        [SettingSource("Show difficulty graph", "Whether a graph should show difficulty throughout the beatmap")]
```

Comment:
Not sure if just me but the description here seems to be worded a touch funny... If I were to suggest anything:

```suggestion
        [SettingSource("Show difficulty graph", "Whether a graph displaying difficulty throughout the beatmap should be shown")]
```